docs/ref/models/instances.txt

```diff
@@ -834,6 +834,15 @@ Note that in the case of identical date values, these methods will use the
 primary key as a tie-breaker. This guarantees that no records are skipped or
 duplicated. That also means you cannot use those methods on unsaved objects.
 
+.. admonition:: Overriding extra instance methods
+
+    In simple cases overriding or inheriting ``get_FOO_display()``,
+    ``get_next_by_FOO()``, and ``get_previous_by_FOO()` should work as
+    expected. Since they are added by the metaclass, however, it is not
```

Comment:
`     expected, since they are added by the metaclass. However, it is not `

Reply:
This one reverses the intended meaning. 

I want to say that the reason we can't handle all cases is precisely because the methods are added by the meta class. (Not that they work because they're added...)

Reply:
Ahh, OK.